File: src/main/java/com/example/samla/userinterface/ToastManager.java
package com.example.samla.userinterface;

import android.app.Activity;
import android.widget.Toast;

public class ToastManager {
    public static void makeToast (final Activity activity, final String message) {
        activity.runOnUiThread(new Runnable() {
            public void run() {
                Toast.makeText(activity, message, Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: src/main/java/com/samla/sdk/userinterface/ToastManager.kt
package com.samla.sdk.userinterface

import android.app.Activity
import android.widget.Toast

object ToastManager {
    @JvmStatic
    fun makeToast(activity: Activity, message: String?) {
        activity.runOnUiThread { Toast.makeText(activity, message, Toast.LENGTH_SHORT).show() }
    }
}